src/mockApi/authUser.js

```diff
@@ -0,0 +1,6 @@
+import { getAuth, signInWithEmailAndPassword } from 'firebase/auth';
+
+export function auth(app) {
+  const authApp = getAuth(app);
+  signInWithEmailAndPassword(authApp, 'then7w@gmai.com', 'Angland2010').then((response) => console.log(response));
```

Comment:
These are probably sensitive and also hardcoded variables. Keep them separately to switch them easily.

Reply:
Ok, now it is not needed. This is I tested Firebase. Deleted.